Tests/ReactiveSwiftTests/PropertySpec.swift

```diff
@@ -285,7 +285,7 @@ class PropertySpec: QuickSpec {
 
 					target! <~ source
 
-					queue.async(group: group) {
+					queue.async(group: group, flags: .barrier) {
```

Comment:
It limits the concurrency, but IIRC does not affect the assertion here which was about the mitigated data race between the two blocks, not across iterations.